relation_engine_server/api_versions/api_v1.py

```diff
@@ -195,6 +196,15 @@ def show_config():
     )
 
 
+@api_v1.route("/ensure_specs", methods=["GET"])
+def ensure_all_specs():
+    failed_names = ensure_specs.ensure_all()
+    if any([name for schema_type, names in failed_names.items() for name in names]):
```

Comment:
Take a look at the documentation for `ensure_specs.ensure_all`

Reply:
Would you like me to document everything in this function as well?

Reply:
The `names` is a list of `name`s of local specs that failed. The `name`s are made by the concatenation of the required fields in the local specs.

Reply:
Can you change the docs to something more like this:

```
Ensure that all the specs in the spec/**/*.json and spec/**/*.yaml are present on the server. This endpoint can be used to ensure the integrity of the server specs without redeployment.
```

If you write something about the endpoint not being necessary, it makes the user wonder why it got left in.

Reply:
I already added some documentation explaining the cruftiness of this function in a previous commit.
```
    """
    This endpoint is not strictly necessary, as the ensure_specs.ensure_all()
    code should triggered in startup scripts. This is more insurance in case
    one wishes to ensure the specs without re-deployment
    """
```

Reply:
As an API consumer, I want to know what the function does and what it returns, ideally without looking through other files to find the function that it calls. The current documentation does not do either of those things.

Reply:
Yes, I suppose since this is an API the docs should already be consistently explicit. The documentation should now do either of those things, for my function at least